files/sinonjs/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "npm",
+  "name": "sinon",
+  "repo": "sinonjs/sinon",
+  "files": {
+    "basePath": "lib/",
+    "include": ["sinon.js"]
```

Comment:
As far as I see its not possible. The authors are not publishing the production files anywhere where our bot could take them. Sorry